doc/user/content/get-started.md

```diff
@@ -73,237 +62,465 @@ Download and install the [PostgreSQL installer](https://www.postgresql.org/downl
+
+1. Use the [`SHOW CLUSTER REPLICAS`](https://materialize.com/docs/sql/show-cluster-replicas/) command to check the status of the replicas:
 
     ```sql
-    CREATE CONNECTION kafka_connection TO KAFKA (
-      BROKER 'pkc-n00kk.us-east-1.aws.confluent.cloud:9092',
-      SASL MECHANISMS = 'PLAIN',
-      SASL USERNAME = SECRET kafka_user,
-      SASL PASSWORD = SECRET kafka_password
-    );
+    SHOW CLUSTER REPLICAS WHERE CLUSTER = 'compute_qck';
+    ```
+
+    ```noftm
+       cluster   | replica |  size   | ready
+    -------------+---------+---------+-------
+     compute_qck | r1      | 2xsmall | t
+    (2 rows)
     ```
 
-    In addition to Kafka, we need a connection to a Schema Registry that Materialize will use to fetch the schema of our sample data.
+    Once the `r1` replica is ready (`ready=t`), move on to the next step.
+
+## Step 3. Ingest streaming data
+
+Materialize supports streaming data from multiple external sources, including [Kafka](/sql/create-source/kafka/) and [PostgreSQL](/sql/create-source/postgres/). The process for integrating a source typically involves configuring the source's network and creating connection objects in Materialize.
+
+For this guide, you'll keep it simple and use a [built-in load generator](/sql/create-source/load-generator/#auction) that simulates an auction house, where users bid on an ongoing series of auctions.
+
+1. Use the [`CREATE SOURCE`](/sql/create-source/load-generator/) command to create the auction house source:
 
     ```sql
-    CREATE CONNECTION csr_connection TO CONFLUENT SCHEMA REGISTRY (
-      URL 'https://psrc-ko92v.us-east-2.aws.confluent.cloud:443',
-      USERNAME = SECRET csr_user,
-      PASSWORD = SECRET csr_password
-    );
+    CREATE SOURCE auction_house
+      IN CLUSTER ingest_qck
+      FROM LOAD GENERATOR AUCTION (TICK INTERVAL '50ms')
+      FOR ALL TABLES;
     ```
 
-    Once created, these connections can be reused across multiple `CREATE SOURCE` statements.
+    Note that the `IN CLUSTER` clause attaches this source to the existing `ingest_qck` cluster, but it's also possible to create a cluster and replica at the time of source creation using the [`WITH SIZE`](/sql/create-source/load-generator/#with-options) option.
 
-1. **Create sources.** Now that we have the details to connect, we can create a source for each Kafka topic we want to use.
+1. With the source created, Materialize starts ingesting data into durable storage, automatically splitting the stream into multiple _subsources_ that represent different tables. Use the [`SHOW SOURCES`](/sql/show-sources/) command to get an idea of the data being generated:
 
     ```sql
-    CREATE SOURCE purchases
-      FROM KAFKA CONNECTION kafka_connection (TOPIC 'mysql.shop.purchases')
-      FORMAT AVRO USING CONFLUENT SCHEMA REGISTRY CONNECTION csr_connection
-      ENVELOPE DEBEZIUM
-      WITH (SIZE = '3xsmall');
+    SHOW SOURCES;
```

Comment:
Same question around the terminal command indicator, since we're using `psql` to interact with MZ, it might be helpful to new users to differentiate from a CLI and psql command. `materialize=>` for the command prompt

Reply:
This doesn't work today. Again, would love you to love this as a tooling improvement.